www_src/components/action-menu/action-menu.less

```diff
@@ -0,0 +1,81 @@
+/*********************************************************
+* Action Menu
+*/
+.action-menu {
+  @_bottomGutter: 10px;
+
+  @_primaryBg: #69A0FC;
+  @_primaryText: #FFF;
+  @_secondaryBg: #FFF;
+  @_secondaryText: #718095;
+
+  @_primarySize: 56px;
+  @_secondarySize: 40px;
+  @_spacing: 56px;
+
+  position: fixed;
+  left: 50%;
+  bottom: @_bottomGutter;
+  z-index: 999;
+  width: @_primarySize;
+  margin-left: -(@_primarySize/2);
```

Comment:
in this case i think it's better to do this like this since using flexbox would require a large container element at full width, which will block content behind it